src/components/pages/PendingApproval/RenderPendingApproval.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import 'antd/dist/antd.css';
+import '../Dashboard/Admin/index.css';
+import { Layout, Menu, Breadcrumb } from 'antd';
+import { Calendar } from 'antd';
+
+import {
+  AppstoreOutlined,
+  LaptopOutlined,
+  LineChartOutlined,
+  TeamOutlined,
+  UserOutlined,
+  CalendarOutlined,
+  FormOutlined,
+} from '@ant-design/icons';
+
+const { SubMenu } = Menu;
+const { Header, Content, Sider } = Layout;
+
+const RenderPendingApproval = props => {
+  return (
+    <Layout>
+      {/* <Header className="header">
+          <div className="logo" />
+          <Menu className="theme" mode="horizontal" defaultSelectedKeys={['2']}>
+            <Menu.Item key="1">nav 1</Menu.Item>
+            <Menu.Item key="2">nav 2</Menu.Item>
+            <Menu.Item key="3">nav 3</Menu.Item>
+          </Menu>
+        </Header> */}
```

Comment:
Not merge-stopping, but we should always try to remove code rather than leave it commented out!